packages/core/src/components/nav/nav-utils.ts

```diff
@@ -70,17 +70,15 @@ export function updateZIndex(viewController: ViewController, newZIndex: number)
   }
 }
 
-export function toggleHidden(element: HTMLElement, isVisible: Boolean, shouldBeVisible: boolean) {
-  if (isVisible !== shouldBeVisible) {
-    element.hidden = shouldBeVisible;
-  }
+export function toggleHidden(element: HTMLElement, shouldBeHidden: boolean) {
+    element.hidden = shouldBeHidden;
 }
 
-export function canNavGoBack(nav: Nav) {
+export function canNavGoBack(nav: Nav, view?: ViewController) {
   if (!nav) {
     return false;
   }
-  return !!nav.getPrevious();
```

Comment:
this function returns a boolean, we need the cast to boolean, right?

Reply:
@mhartington - I think Manu's right